CHANGELOG.md

```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
 - Clean up code by using Swift 5.7 capabilities. 
 - `.pagerTabItem(tag: SelectionType)` modifier now requires tag parameter which identifies the pager tab item. 
 - `pagerTabStripViewStyle(_ style: PagerStyle)` is the new modifier to set up pager style. 
-- Selection value, which indicates wich is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
-- Fix scroll behaivour when pager is added inside another pager or scrllable container. Whenever child page reatch its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
```

Comment:
The Selection value indicating the selected page now supports any Hashable value, not just an Int as in the previous version.